thesis-de/content-de.tex

```diff
@@ -1492,6 +1479,26 @@ \subsubsection{Vorbereitung der Evaluierung}
   hier unabhängig von kostenpflichtigen Lizenzen und Copyright Rechten zu sein.
   Ebenfalls ist eine Open Source Software nötig, da hier der Quelltext der
   Software angepasst und verändert werden darf.
+  \1 Bei der Evaluierung der Weboberfläche, sollte zusätzlich zu den bereits 
```

Comment:
die einleitung ist redundant. wir sind bereits in der liste der Kriterien, das muss nicht explizit erwähnt werden.